### DISCUSSION TITLE
Millenium Toms

### DISCUSSION THREAD

**Discussion by 3hhh** [corrados/edrumulus#80] [Q&A] [answered]
Hi all,

I just had a look inside a Millenium MPS 750x Tom and noticed that it has 2 Piezos:
One in the center for rimshots and one at the top for regular hits.

If I understand this project correctly, it requires a center-mounted piezo for positional sensing, i.e. those toms shouldn't be compatible.

However it seems that the piezos could be switched rather easily (M2 15mm plastic distance tubes + screws should probably do it).

Should that work? Did anyone possibly try it already?

Thanks in advance for any responses!

**Comment by corrados** [comment]
I would not hit directly on the piezo. The Roland center mounted piezos have a foam cone but on the MPS750x tom it seems that the piezo is directly mounted on the mesh hat. Is that correct? Maybe you could post an image of the inside of the pad.

**Comment by 3hhh** [comment]
The top piezo (i.e. the one apparently responsible for regular hits) has some foam on top of it, too. I guess I'll give it a try and see if I can switch them...

![1E4Nw4n_Z3nFjXQZgaCQ](https://github.com/corrados/edrumulus/assets/13033030/ec75f109-cadb-48c7-bd3e-7bb3fbb92d50)

**Reply by corrados** [reply]
Thanks for the picture. The diameter of the foam is quite large. Roland also use such sized for piezos mounted at the edge of the pad. For center mounted piezos, its a cone shape where the diameter of the foam touching the head is much smaller. I assume that this is to minimize the hot-spot issue and also it seems to be beneficial for positional sensing.

I cannot recommend to move the rim piezo to the side. From my experience it is important for this sensor to be placed in the middle of the pad. E.g., the old Roland PD-120 has the rim piezo mounted not in the middle and the rim shot detection performance is very bad. Modern Roland pads have them in the middle (as in your picture).

**Reply by 3hhh** [reply]
Thanks for the insight!

So Roland places two piezos on top of each other with some distance tubes in between?

I could try that, too (the center piezo fits below, if necessary).

However my assumption that edrumulus requires center mounted piezos is right, is it not?
Otherwise my endeavour is rather pointless after all...

**Reply by corrados** [reply, marked as answer]
In [this image](https://ibb.co/pJyX3Jq) (or [this image](https://ibb.co/WKDHZ3j)) you can see how Roland does it. And yes, if you want to have positional sensing, you'll need a center mounted pizeo.

**Reply by 3hhh** [reply]
Interesting. So they stack them above each other and try to decouple them with some foam between the screws.
I guess I'll try to replicate that once I get the distance tubes...

**Reply by 3hhh** [reply]
The test with center piezos in combination with the Millenium sound computer was a success - I didn't notice any difference in behaviour so far. In fact it might even be better than before... I still have to test it with edrumulus though.

Required hardware per tom:
- 4x M3 distance bolts, 12mm height, female-female
- 8x M3 screws, 5-6mm height
- some additional foam for the snare as the piezo needs some more height (is included with the MPS 750x drum set) or just replace the trigger cushion entirely (18mm height, 35mm diameter)

Cutting the foam to [Roland-like cones](https://ibb.co/pJyX3Jq) appears to avoid accidental rim shots caused by hitting the center of the pad.

**Reply by 3hhh** [reply]
Further discussion can be found here: https://github.com/corrados/edrumulus/discussions/81

**Reply by 3hhh** [reply]
By now I'd not recommend this modification anymore for the following reasons:
1. You only need it for positional sensing anyway and usually _only_ on the snare. I couldn't get it to work reliably on the MPS 750x snare though, i.e. it's rather pointless there, too.
2. Having the piezo in the middle of the pad may enable positional sensing, but also increases the hot spot effect there (hits there are generally louder where the piezo is), i.e. it even produces a small disadvantage wrt dynamics.

**Reply by 3hhh** [reply]
I tried it again on the snare and the hot spot effect appears to be less strong and pos sensing appears to work okayish (on one side it doesn't, which is kind of strange).
IIRC there were some changes to edrumulus hot spot handling in the recent commits, which may have helped with the hot spot effect.
Moreover I used a very soft cushion.
